src/main/java/appeng/container/slot/SlotCraftingTerm.java

```diff
@@ -153,6 +157,40 @@ else if( action == InventoryAction.CRAFT_STACK ) // craft into hand, full stack
 		}
 	}
 
+	// TODO: This is really hacky and NEEDS to be solved with a full container/gui refactoring.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)